docs/reference/configuration/keywords.rst

```diff
@@ -83,21 +80,66 @@ Keyword name                                                            Required
 
 
 
-Basic required keywords
+Commonly used keywords
 -----------------------
-.. _basic_required_keywords:
+.. _commonly_used_keywords:
+
+.. _num_realizations:
+.. topic:: NUM_REALIZATIONS
+
+        This is the size of the ensemble, i.e. the number of
+        realizations/members in the ensemble. All configs must contain this
+        keyword.
+
+        *Example:*
+
+        ::
+
+                -- Use 200 realizations/members
+                NUM_REALIZATIONS 200
+
+.. _define:
+.. topic:: DEFINE
+
+        With the DEFINE keyword you can define key-value pairs which will be
+        substituted in the rest of the configuration file. The DEFINE keyword expects
+        two arguments: a key and a value to replace for that key. Later instances of
+        the key enclosed in '<' and '>' will be substituted with the value. The value
+        can consist of several strings, in that case they will be joined by one single
```

Comment:
Is this how the DEFINE keyword works now? 

```
The DEFINE keyword expects
        two arguments: a key and a value to replace for that key. Later instances of
        the key enclosed in '<' and '>' will be substituted with the value.
```
Is this so, because we secretly hate our users and want to make their life difficult? Also, why is the `suggestor` than suggesting: 

```python
  for key, _ in defines:
            if key.count("<") + key.count(">") != 2 or key[0] != "<" or key[-1] != ">":
                _suggestion = (
                    "Using DEFINE or DATA_KW with substitution"
                    " strings that are not of the form '<KEY>' is deprecated"
                    " and can result in undefined behavior. "
                    f"Please change {key} to <{key.replace('<', '').replace('>', '')}>"
                )
                logger.info("Deprecated use of DEFINE without <>: %s", _suggestion)
```

Reply:
Fixed.